plugins/janus_sip.c

```diff
@@ -3612,6 +3612,27 @@ static void *janus_sip_handler(void *data) {
 
 			if(offer)
 				session->sdp->o_version++;
+
+			gboolean audio_added = strstr(msg_sdp, "m=audio") && !strstr(msg_sdp, "m=audio 0") && session->media.local_audio_rtp_port == 0;
+			gboolean video_added = strstr(msg_sdp, "m=video") && !strstr(msg_sdp, "m=video 0") && session->media.local_video_rtp_port == 0;
+
+			if(audio_added)
+				session->media.has_audio = TRUE;	/* FIXME Maybe we need a better way to signal this */
+
+			if(video_added)
+				session->media.has_video = TRUE;	/* FIXME Maybe we need a better way to signal this */
+
+			if(audio_added || video_added) {
```

Comment:
I think you can remove the three empty lines above, since they belong to the same group of code.